src/components/Questionnaire/TextInput.tsx

```diff
@@ -70,8 +73,13 @@ const StyledOutlinedInput = styled(OutlinedInput)(() => ({
   "&.MuiInputBase-multiline": {
     padding: "12px",
   },
+  "& .MuiInputBase-inputMultiline": {
```

Comment:
If you specify `multiline` but don't provide `rows`, we get styling with `NaNpx` values. Whether you want to address this or not is your call, but I just wanted to point it out.

![Screenshot 2024-05-13 at 2 49 33 PM](https://github.com/CBIIT/crdc-datahub-ui/assets/38357871/8d5b0cf5-c0d0-4293-9b69-764f6eea2332)

Reply:
Updated to default to 1 * row height instead.